packages/template-mrt-reference-app/app/ssr.js

```diff
@@ -207,6 +208,20 @@ const responseHeadersTest = async (req, res) => {
     res.json(jsonFromRequest(req))
 }
 
+const ssrShared = async (req, res) => {
```

Comment:
This code works locally but gets a server error when running in MRT. Needs more logging to debug.